tests/integration/smb_share_test.go

```diff
@@ -85,7 +85,12 @@ func (s *SmbShareSuite) getPodIP() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return pods[0].Status.PodIP, nil
```

Comment:
IMO, the name of the function getPodIP implies that you expect to find one pod only and get its IP. This is however finding one ready pod of many and returning its IP and that feels wrong.